Week3/homework/js-exercises/giveCompliment.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+function giveCompliment(name){
+   let compliment = [ 'good','awesome','strong','smart','handsome','clever','nice','perfect','kind','something']
+console.log('You are ', compliment[Math.floor(Math.random()*10) ]+','  , name);
```

Comment:
Here the same question, what if the array is bigger (or smaller) than 10? Does this still work?

Reply:
no, it will not work, it will only run till first element.
should i make it work without *10?